altair/vegalite/v5/api.py

```diff
@@ -96,7 +127,24 @@
         ResolveMode_T,
     )
 
+
 ChartDataType: TypeAlias = Optional[Union[DataType, core.Data, str, core.Generator]]
+_TSchemaBase = TypeVar("_TSchemaBase", bound=core.SchemaBase)
+_T = TypeVar("_T")
+_OneOrSeq = TypeAliasType("_OneOrSeq", Union[_T, Sequence[_T]], type_params=(_T,))
+"""One of ``_T`` specified type(s), or a `Sequence` of such.
+
+Examples
+--------
+The parameters ``short``, ``long`` accept the same range of types::
+
+    # ruff: noqa: UP006, UP007
+
+    def func(
+        short: _OneOrSeq[str | bool | float],
```

Comment:
I'll be moving this into `_typing.py` eventually, but wanted to leave the trail of the origin in [`3fa4a3f` (#3427)](https://github.com/vega/altair/pull/3427/commits/3fa4a3f7baa26e7ea4c6eacac778b91a9801bd58)